src/pages/App.tsx

```diff
@@ -86,8 +87,10 @@ export default function App() {
             <Polling />
             <TopLevelModals />
             <Switch>
-              <Route exact strict path="/vote" component={Vote} />
-              <Route exact strict path="/vote/:governorIndex/:id" component={VotePage} />
+              <Suspense fallback={<Loader />}>
```

Comment:
I'm reusing the `<Loader />` component here.

Reply:
@callil is this an appropriate loading element?